src/main/java/book/store/service/impl/UserServiceImpl.java

```diff
@@ -13,8 +15,11 @@
 @Service
 @RequiredArgsConstructor
 public class UserServiceImpl implements UserService {
+    private static final Long USER_ROLE_ID = 1L;
```

Comment:
Why do you hardcoded the value of role id?

Reply:
To get a role from DB ("roles" table) by id. This is "ROLE_USER" with id 1L, and it is default for any new registered person.

Reply:
remove this hardcode, please
fetch Role by name using enum